guides/private-app/src/main.ts

```diff
@@ -1,21 +1,8 @@
-import './style.css'
```

Comment:
Keep this because it works like this in other places like public-app and templates. And I've a step by step guide that I will need to change